src/commands/edge_app_utils.rs

```diff
@@ -308,9 +298,43 @@ mod tests {
         assert!(result.is_ok());
         let changes = result.unwrap();
         assert!(changes.changes_detected);
-        assert_eq!(changes.creates.len(), 2);
+        assert_eq!(changes.creates.len(), 1);
+        assert_eq!(changes.creates[0].title, "google_maps_api_key");
     }
 
+    // TODO: Update test, when patching is implemented
```

Comment:
can't you disable it instead? I really hate commented out code

Reply:
ok, though I wanted to implement it in next PR

Reply:
Disabled